src/ModuleConfig.php

```diff
@@ -22,7 +22,6 @@ public function __invoke()
                     'graphql.processor' => ProcessorFactory::class,
                     'graphql.schema'    => Schema\SchemaFactory::class,
                     GraphQLMiddleware::class => GraphQLMiddlewareFactory::class,
-                    GraphQLMiddlewareDelegatorFactory::class => GraphQLMiddlewareDelegatorFactory::class
```

Comment:
because it's useless (at least in my experience), and it fails : 

> PHP Catchable fatal error:  Argument 3 passed to GraphQLMiddleware\GraphQLMiddlewareDelegatorFactory::__invoke() must be callable, string given in /home/.../vendor/stefanorg/graphql-middleware/src/GraphQLMiddlewareDelegatorFactory.php on line 24